articles/azure-functions/functions-monitoring.md

```diff
@@ -432,11 +432,7 @@ namespace functionapp0915
 
 Don't call `TrackRequest` or `StartOperation<RequestTelemetry>`, because you'll see duplicate requests for a function invocation.  The Functions runtime automatically tracks requests.
 
-Set `telemetry.Context.Operation.Id` to the invocation ID each time your function is started. This makes it possible to correlate all telemetry items for a given function invocation.
-
-```cs
-telemetry.Context.Operation.Id = context.InvocationId.ToString();
-```
+Don't set `telemetryClient.Context.Operation.Id`. This is a global setting and will cause incorrect correllation when many functions are running simultaneously. Instead, create new telemetry instances (`DependencyTelemetry`, `EventTelemetry`, etc), modify the `Context` properties of those objects, and pass them as parameters to the corresponding `Track` method on `TelemetryClient` (`TrackDependency()`, `TrackEvent()`, etc). This ensures that the telemetry has the correct correllation details for the current function invocation.
```

Comment:
This is clear but style guidelines prohibit abbreviations such as "etc" and the sentence that contains it is long. In this instance I think you can omit etc. Could rephrase something like 

> Instead, create a new telemetry instance (`DependencyTelemetry`, `EventTelemetry`) and modify its `Context` property. Then pass in the telemetry instance to the corresponding `Track` method on `TelemetryClient` (`TrackDependency()`, `TrackEvent()`).

Reply:
That sounds better. I'll update.